packages/pages/src/dev/dev.ts

```diff
@@ -6,10 +6,11 @@ import { ProjectFilepaths } from "../common/src/project/structure.js";
 
 interface DevArgs extends Pick<ProjectFilepaths, "scope"> {
   local?: boolean;
+  "prod-url"?: boolean;
```

Comment:
remove quotes

Reply:
you need quotes here because of the dash!